launcher/launcher.go

```diff
@@ -787,6 +787,16 @@ nextNetInstance:
 			}
 		}
 
+		uid, err := launcher.storage.GetInstanceUID(netInstance)
```

Comment:
Removing UID doesn't relate to `removeInstanceNetworkParameters`. IMHO it should not be here.
Also, tests failed on CI.

Reply:
Done